src/views/Cart/scss/index.scss

```diff
@@ -49,6 +49,11 @@
   }
 
   &__thumbnail {
+    a {
+      display: table-cell;
```

Comment:
This doesn't look right. Why would you use `display: table-cell` on a component inside `<td />`? I think we should use flex here